lib/coustom_libs/database.dart

```diff
@@ -19,28 +20,30 @@ class Databaseobj{
 
   Databaseobj(this.context);
 
-  write(mainchapter, chapter, subchapter, resultlist){
-    int i = 0;
-    for(var result in resultlist){
-
-      // List list = DatabaseWidget.of(context).database.get("[$mainchapter][$chapter][${subchapter[0]}]");
-      //print("liste :: $list");
-      try{  
-        List list = DatabaseWidget.of(context).prog_database.get(subchapter.length == 0  ? "[$mainchapter][$chapter]" : "[$mainchapter][$chapter][${subchapter[i]}]");
-        int x = 0;
-        List<dynamic> updatedres = list.map((item){x++; return  item + result[x - 1];}).toList();
-        DatabaseWidget.of(context).prog_database.put(
-            subchapter.length == 0  ? "[$mainchapter][$chapter]" : "[$mainchapter][$chapter][${subchapter[i]}]",
-            updatedres
-          );
-      }catch(e){
-        DatabaseWidget.of(context).prog_database.put(
-        subchapter.length == 0  ? "[$mainchapter][$chapter]" :"[$mainchapter][$chapter][${subchapter[i]}]",
-        (result as List<dynamic>)
-        );
-      }
-      i++;
-    }
+  void write(QuestionController questionController, BuildContext context){
+    
+
+    // int i = 0;
```

Comment:
Warum toter code?

Reply:
Brauche ich noch beim rebase. Muss ich alles neu implementieren weil ich objektorientiert die Fragen mit extra Daten speichern möchte. Der PR ist noch work in progress. Wir müssen davon ab auch schauen welche Daten wir brauchen insbesondere zur Synchronisation über 50ohm.de.